coveralls-ruby.gemspec

```diff
@@ -20,9 +20,9 @@ Gem::Specification.new do |gem|
   gem.required_ruby_version = '>= 1.8.7'
 
   gem.add_dependency 'json', '>= 1.8', '< 3'
-  gem.add_dependency 'simplecov', '~> 0.12.0'
-  gem.add_dependency 'tins', '~> 1.6'
-  gem.add_dependency 'term-ansicolor', '~> 1.3'
+  gem.add_dependency 'simplecov', '~> 0.10.0'
```

Comment:
It's all about CI which teletransport you to and old past. I wonder if this can be done only if you're using ruby 1.9